nautobot/extras/forms/forms.py

```diff
@@ -848,6 +852,104 @@ def clean(self):
         return cleaned_data
 
 
+class JobBulkEditForm(NautobotBulkEditForm):
+    """Bulk edit form for `Job` objects."""
+
+    pk = forms.ModelMultipleChoiceField(
+        queryset=Job.objects.all(),
+        widget=forms.MultipleHiddenInput(),
+    )
+    grouping = forms.CharField(
+        required=False,
+        help_text="Human-readable grouping that this job belongs to",
+    )
+    description = forms.CharField(
+        max_length=200,
+        required=False,
+        help_text="Markdown formatting and a limited subset of HTML are supported",
+    )
+    enabled = forms.NullBooleanField(
+        required=False, widget=BulkEditNullBooleanSelect, help_text="Whether this job can be executed by users"
+    )
+    has_sensitive_variables = forms.NullBooleanField(
+        required=False, widget=BulkEditNullBooleanSelect, help_text="Whether this job contains sensitive variables"
+    )
+    approval_required = forms.NullBooleanField(
+        required=False,
+        widget=BulkEditNullBooleanSelect,
+        help_text="Whether the job requires approval from another user before running",
+    )
+    hidden = forms.NullBooleanField(
+        required=False,
+        widget=BulkEditNullBooleanSelect,
+        help_text="Whether the job defaults to not being shown in the UI",
+    )
+    dryrun_default = forms.NullBooleanField(
+        required=False,
+        widget=BulkEditNullBooleanSelect,
+        help_text="Whether the job defaults to running with dryrun argument set to true",
+    )
+    soft_time_limit = forms.FloatField(
+        required=False,
+        validators=[MinValueValidator(0)],
+        help_text="Maximum runtime in seconds before the job will receive a <code>SoftTimeLimitExceeded</code> "
+        "exception.<br>Set to 0 to use Nautobot system default",
+    )
+    time_limit = forms.FloatField(
+        required=False,
+        validators=[MinValueValidator(0)],
+        help_text="Maximum runtime in seconds before the job will be forcibly terminated."
+        "<br>Set to 0 to use Nautobot system default",
+    )
+    task_queues = JSONArrayFormField(
+        base_field=models.CharField(max_length=100, blank=True),
+        help_text="Comma separated list of task queues that this job can run on. A blank list will use the default queue",
+        required=False,
+    )
+    # Flags to indicate whether the above properties are inherited from the source code or overridden by the database
+    # Text field overrides
+    clear_grouping_override = forms.BooleanField(
+        required=False,
+        help_text="If set, the configured grouping will be reverted back to the default value from the Job source code",
```

Comment:
Perhaps something along these lines might be better?

```suggestion
        help_text="If checked, groupings will be reverted to the default values defined in each Job's source code",
```